<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowAsDlg</class>
 <widget class="QDialog" name="ShowAsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show As</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Show as date(M/D/Y):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QComboBox" name="comboBox_Month"/>
   <widget class="QComboBox" name="comboBox_Day"/>
   <widget class="QComboBox" name="comboBox_Year"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
